### ISSUE TITLE
Vendor review for premailer

### ISSUE THREAD

**Issue by davidascher** [jbuck/mofo-architecture#1]
Do we need to do a vendor review for premailer?  /cc @simonwex @adamlofting @jbuck

**Comment by adamlofting** [comment]
Yes, if premailer is a hosted service and we're pushing any user data through it, it will need a vendor review. 

If we don't have one already, this is the form to kick off the review: https://bugzilla.mozilla.org/enter_bug.cgi?product=Data%20Compliance
